cabal.project

```diff
@@ -63,6 +63,7 @@ constraints:
   -- remove this when hlint sets ghc-lib to true by default
   -- https://github.com/ndmitchell/hlint/issues/1376
   hlint +ghc-lib,
+  ghc-check -ghc-check-use-package-abis,
```

Comment:
@wz1000 could you add a comment explaining this on master? It's a bit mysterious otherwise.